test_tracetools/CMakeLists.txt

```diff
@@ -232,6 +232,7 @@ if(BUILD_TESTING)
       rmw_connextdds
       rmw_cyclonedds_cpp
       rmw_fastrtps_cpp
+      rmw_fastrtps_dynamic_cpp
```

Comment:
We discussed using `get_available_rmw_implementations()` in https://github.com/ros2/ros2_tracing/pull/116#discussion_r1645042581. However, that would mean that any new tier 1 `rmw` implementation (or any `rmw` implementation in a local workspace) would need to be instrumented for the `test_tracetools` tests to pass. I don't think we want to impose this constraint.

Reply:
agree with you on this, i think current list is fine for now.